libr/core/core.c

```diff
@@ -1276,7 +1277,11 @@ R_API int r_core_init(RCore *core) {
 
 	r_core_config_init (core);
 
+	ut64 diff, prev = r_sys_now();
 	r_core_loadlibs_init (core);
+	diff = r_sys_now() - prev;
+	sdb_num_add(DB, "r_core_loadlibs_init time", diff, 0);
```

Comment:
maybe it should be `r_core_loadlibs_init_time` I am not sure of if the command line will be able to handle queries with space in the name @radare